docs_user/main.adoc

```diff
@@ -0,0 +1,3 @@
+= Data Plane Adoption
```

Comment:
I wonder if we'll want the OpenStack + Ceph procedures in one AsciiDoc document, or whether we'll want to separate them...

But anyway maybe this is something that should be relatively easy to tweak later if needed.

Reply:
cc @klgill @fmount

Reply:
I want to clarify that I agree with keeping the Ceph procedure within the overall adoption guide, but as its own chapter.  Maybe I misunderstood Jiri. 
That's similar to the current FFU guide. The customer follows the FFU guide in sequence, and follows the Ceph chapter if that applies to their environment.